src/App.test.tsx

```diff
@@ -1,10 +1,10 @@
 import * as React from 'react';
 import App from './App';
-import { shallow } from 'enzyme';
+import { render } from 'enzyme';
 
 it('renders without crashing', () => {
-  const app = shallow(
+  const app = render(
```

Comment:
Won't this render the full app causing the `JamText` component to call out to our backend?

Reply:
Potentially. I believe it only renders the dom but doesn't make the components fully functional, not sure. Either way, this version doesn't have any of the real components in it. So it works for now and once tas tries to merge he will have to deal with it :)

Reply:
Is there another way that will actually render all the components without making backend calls? With the new "Flex" components you don't get any content with the "shallow" command. But maybe that is another reason to move away from the react-flex library.